static/images/conference/index/2018/d

```diff
@@ -0,0 +1 @@
+
```

Comment:
Looks like an extra file got created, it should be removed. Also, not sure how, but the banner image is too small a resolution now (it's only about 400px wide). However the new crop looks correct. So I've cropped the original to match the one below - you should use this one as it's still 1024px wide.
![title-banner](https://user-images.githubusercontent.com/399209/33848329-602db27c-de7b-11e7-9eba-1762b7b7f2da.jpg)

Reply:
Oops lol the extra file there was for me to create the folder again. I'll go remove it and add the new image.